src/cmds/manifest/manifest.ts

```diff
@@ -72,17 +77,20 @@ module.exports = async function manifest(argv) {
       bundle.url = `${LOCALHOST_START}:${port}/${bundlePath}`;
     }
 
-    // sort web3Wallet object keys
-    Object.entries(pkg.web3Wallet).forEach(([k, v]) => {
-      if (typeof v === 'object' && !Array.isArray(v)) {
-        pkg.web3Wallet[k] = Object.keys(v).sort().reduce(
-          (acc, l) => {
-            acc[l] = v[l];
-            return acc;
-          }, {},
-        );
-      }
-    });
+    if (pkg.web3Wallet) {
+      // sort web3Wallet object keys
+      Object.keys(pkg.web3Wallet).sort().forEach((k) => {
+        const v = (pkg.web3Wallet as unknown as Wallet)[k];
+        if (typeof v === 'object' && !Array.isArray(v) && v !== null) {
+          (pkg.web3Wallet as unknown as Wallet)[k] = Object.keys(v).sort().reduce(
+            (acc, l) => {
+              (acc as any)[l] = (v as any)[l];
+              return acc;
+            }, {},
+          );
+        }
+      });
+    }
```

Comment:
We should change all the single-character variable names to sensible words, and rename `acc` to `propertyValue` or something such.

Reply:
`acc` of course comes from "accumulator", which describes the parameter's purpose in the reducer function, but it's better for legibility for the parameter name to describe its purpose in the context of our program.

In other words, its function as the accumulator of a reducer function is incidental to the work we are trying to accomplish, and it is precisely "the work we are trying to accomplish" that should be communicated by our code for the benefit of others and our future selves.